cms/models.py

```diff
@@ -72,6 +72,42 @@ def save(self, *args, **kwargs):
         super(LearningOutcomesPage, self).save(*args, **kwargs)
 
 
+class ForTeamsPage(CourseProgramChildPage):
+    """
+    CMS Page representing a "For Teams" section in a course/program page
+    """
+
+    content = RichTextField(help_text="The content shown in the section")
+    action_title = models.CharField(
+        max_length=255, help_text="The text to show on the call to action button"
```

Comment:
Optional: for readability you can move properties into separate lines.

Reply:
The formatter being used in the project (black) will not allow breaking into multiple lines unless it extends beyond the standard line length. If we manually do it, the Travis task for the formatter will exit with a non-zero code.